src/main.ts

```diff
@@ -56,6 +56,7 @@ export async function run(): Promise<void> {
     core.startGroup('Cherry picking')
     const result = await gitExecution([
       'cherry-pick',
+      '-x',
```

Comment:
TBH I am not really in favor of this. Reasons:
- No one really looks at the commit message when reviewing the cherry-pick PRs
- Since we are cherry-picking to multiple branches the commit message would go bigger and bigger.

So lets not have it , as its not useful

Reply:
@Gauravtalreja1 Do you have any reason ? else I would remove that.

Reply:
@jyejare this wouldn't appear as commit message, this will be part of commit body, so I don't think it will bother us.
And its best way to know relation of parent commit and cherrypick commits, other than parent PR number,  also devs use this for foreman/katello projects

Reply:
@jyejare I don't see any harm here.